browser/resources/settings/brave_tor_page/brave_tor_snowflake_install_failed_dialog.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
```

Comment:
We now have a CI check that looks for an HTTPS URL here. So it will need to be changed to https://mozilla.org/MPL/2.0/